project-koku/koku
# [COST-3864] batching polling

## Jira Ticket

[COST-3864](https://issues.redhat.com/browse/COST-3864)

## Description

This change will switch to triggering downloads on an hourly cadence with a max batch limit.

## Testing

1. Checkout Branch
2. Restart Koku
3. Load some test data
4. Hit http://localhost:5042/api/cost-management/v1/download/ within 60s of adding your source.
5. Logs should say polling for accounts but no account gets added
6. Hit http://localhost:5042/api/cost-management/v1/download/ outside of 60s
7. Logs should poll for accounts and collect your provider.

## Notes

...
